pod.go

```diff
@@ -280,19 +287,26 @@ type PodConfig struct {
 
 // valid checks that the pod configuration is valid.
 func (podConfig *PodConfig) valid() bool {
-	if _, err := newHypervisor(podConfig.HypervisorType); err != nil {
-		podConfig.HypervisorType = QemuHypervisor
+	if podConfig == nil {
```

Comment:
Can it happen ? I don't think so because you need podConfig to be valid (not nil) to be able to call this function, right ?